### PR TITLE
Patch dynamo on staging

### PR DESCRIPTION
**Pull request by eric-wu**
1) PLFM-1613 - Increase dynamo throughput and descrease worker frequency
2) PLFM-1614 - Add index.html for health check
3) Reduce dynamo tomcat logging

### CODE REVIEW

**Comment by john-hill** [comment]
Looks g

**Comment by john-hill** [comment]
oops

File: lib/lib-dynamo/src/main/java/org/sagebionetworks/dynamo/config/DynamoConfig.java [hunk 1]
```diff
@@ -26,6 +27,9 @@
 		DynamoKey rangeKey = new DynamoKey(DboNodeLineage.RANGE_KEY_NAME, ScalarAttributeType.S);
 		DynamoKeySchema keySchema = new DynamoKeySchema(hashKey, rangeKey);
 		DynamoThroughput throughput = new DynamoThroughput(18L, 10L);
+		if (this.isProdStack()) {
+			throughput = new DynamoThroughput(75L, 50L);
```
Comment (john-hill) [inline lib/lib-dynamo/src/main/java/org/sagebionetworks/dynamo/config/DynamoConfig.java:31]: Are we still planning to do this dynamically?

Comment (eric-wu) [inline lib/lib-dynamo/src/main/java/org/sagebionetworks/dynamo/config/DynamoConfig.java:31]: Yes, dynamically, I plan to do it in the near future.

**Pull request merged**